### ISSUE TITLE
Tree Grid does not work

### ISSUE THREAD

**Issue by bagerman** [akveo/nebular#2487]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [] feature request

### Issue description

**Current behavior:**

The following exception is thrown while trying to integrate TreeGrid component by following example provided in documentation for the Data Source use 
This is html code at line referenced below in stack :   at AllAnalyticsComponent_Template (all-analytics.component.html:5)
`		<table [nbTreeGrid]="source">
`
Exception stack
```ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'nativeElement' of null
TypeError: Cannot read property 'nativeElement' of null
    at new CdkTable (table.js:1265)
    at new NbTable (index.js:18425)
    at new NbTreeGridComponent (index.js:25275)
    at NodeInjectorFactory.NbTreeGridComponent_Factory [as factory] (index.js:25416)
    at getNodeInjectable (core.js:4029)
    at instantiateAllDirectives (core.js:7963)
    at createDirectivesInstances (core.js:7329)
    at Module.ɵɵelementStart (core.js:13901)
    at AllAnalyticsComponent_Template (all-analytics.component.html:5)
    at executeTemplate (core.js:7302)
    at resolvePromise (zone-evergreen.js:798)
    at resolvePromise (zone-evergreen.js:750)
    at zone-evergreen.js:860
    at ZoneDelegate.invokeTask (zone-evergreen.js:399)
    at Object.onInvokeTask (core.js:27424)
    at ZoneDelegate.invokeTask (zone-evergreen.js:398)
    at Zone.runTask (zone-evergreen.js:167)
    at drainMicroTaskQueue (zone-evergreen.js:569)
    at ZoneTask.invokeTask [as invoke] (zone-evergreen.js:484)
    at invokeTask (zone-evergreen.js:1621)
```
**Expected behavior:**

I would expect the TreeGrid to show up with no errors

**Steps to reproduce:**

Copy provided example and try to run

**Related code:**

```
see documentation code
```

### Other information:
I am using Nebular v6 and Angular v1
**npm, node, OS, Browser**
```
<!--
Node, npm: `node --version` and `npm --version`
OS: Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra?)
Browser: Chrome/Safari/Firefox/etc?
-->
```

**Angular, Nebular**
```
   "@akveo/ng2-completer": "^9.0.1",
    "@angular/animations": "^10.0.11",
    "@angular/cdk": "^10.1.3",
    "@angular/common": "^10.0.11",
    "@angular/compiler": "^10.0.11",
    "@angular/core": "^10.0.11",
    "@angular/forms": "^10.0.11",
    "@angular/google-maps": "^9.1.0",
    "@angular/platform-browser": "^10.0.11",
    "@angular/platform-browser-dynamic": "^10.0.11",
    "@angular/router": "^10.0.11",
    "@asymmetrik/ngx-leaflet": "3.0.1",
    "@nebular/auth": "6.0.0",
    "@nebular/eva-icons": "6.0.0",
    "@nebular/security": "6.0.0",
    "@nebular/theme": "6.0.0",

```

**Comment by sefasaid** [comment]
same here

**Comment by vsct-jburet** [comment]
same issue

**Comment by rrruuuyyy** [comment]
same issue

**Comment by sayax** [comment]
same issue

**Comment by yggg** [comment]
There is a breaking change in the Angular CDK introduced in the patch version 10.1.2 (see [issue in cdk repo](https://github.com/angular/components/issues/20422#issue-686374988) for details). To fix the error downgrade to `@angular/cdk@10.1.1`. Once fix released you can upgrade to the latest version of cdk.
Closing as unrelated to Nebular.
